crates/next-dev/tests/integration/turbopack/basic/polyfill/index.js

```diff
@@ -0,0 +1,3 @@
+it("polyfills `global` to `globalThis`", () => {
+  expect(global).toEqual(globalThis);
```

Comment:
```suggestion
  expect(global).toBe(globalThis);
```

We want to check for same identity